.github/workflows/publish_release_images.yaml

```diff
@@ -0,0 +1,34 @@
+name: Publish Release OCI Images
+
+on:
+  push:
+    branches:
+      - master
```

Comment:
I plan to remove this after we get some release images published to docker hub.

Reply:
Should we have a branch for each release?

Reply:
Just saw this question: what would be the benefit of a branch for each release?